mainwindow.cpp

```diff
@@ -1,27 +1,81 @@
 #include "mainwindow.h"
 #include "ui_mainwindow.h"
 
+#include <QPixmap>
+#include <QRegularExpression>
+#include <QMessageBox>
+
+#include "searchrequest.h"
+
 void CenterWindow(QWidget *widget);
+void showCard(QString path, QLabel* label);
 
 constexpr size_t windowWight = 1200; //px
-constexpr size_t windowHeight = 700; //px
+constexpr size_t windowHeight = 800; //px
+
+void CenterWindow(QWidget *widget);
 
 MainWindow::MainWindow(QWidget *parent)
     : QMainWindow(parent)
-    , ui(std::make_unique<Ui::MainWindow>())
+    , ui(new Ui::MainWindow)
 {
     ui->setupUi(this);
 
+
+    //ui->groupBox_2->setStyleSheet("QFrame{border:0px solid black}");//Hide a card borders
+
+    DisplayCards();
     setFixedSize(windowWight,windowHeight);
 
+    setFixedSize(windowWight,windowHeight);
     CenterWindow(this);
 
+
 }
 MainWindow::~MainWindow()
 {
 }
 
+
+void MainWindow::on_Szukaj_B_clicked(){
+    std::bitset<4> type;
+    type.set(1,true);
+      int zdrowie_od=ui->Zdrowie_od->text().toInt();
+      int zdrowie_do=ui->Zdrowie_do->text().toInt();
+      int zdrowie_value=ui->Zdrowie_value->text().toInt();
+      int koszt_od=ui->Koszt_od->text().toInt();
+      int koszt_do=ui->Koszt_do->text().toInt();
+      int koszt_value=ui->Koszt_value->text().toInt();
+      int atak_od=ui->Atak_od->text().toInt();
+      int atak_do=ui->Atak_do->text().toInt();
+      int atak_value=ui->Atak_value->text().toInt();
+//    (ui->Typ->findChildren<QCheckBox*>("Typ_1",Qt::FindChildrenRecursively));
+    SearchRequest request(zdrowie_od,zdrowie_do,zdrowie_value,
+                          koszt_od,koszt_do,koszt_value,
+                          atak_od,atak_do,atak_value,
+                          ui->Nazwa_T->text().toStdString(),
+                          000,
+                          /*typ*/ 0000,000'000'000'000'0);
```

Comment:
czy są te zera? Przydałaby się do nich jakas nazwa

Reply:
One zostaly zamienione to byly testy..